Update EUICC CSP with latest changes (#9138)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/euiccs-csp.md

Supported operation is Get. Value type is boolean.

<a href="" id="euicc-isactive"></a>**_eUICC_/PPR1Allowed**  

> Required. Indicates whether the download of a profile with PPR1 is allowed. If the eUICC already has a profile (regardless of its origin and policy rules associated with it), then the download of a profile with PPR1 is not allowed.

Reviewer: I'd suggest that the first time it appears we define the acronym, e.g., say "Profile Policy Rule 1 (PPR1)"

> Required. Indicates whether the download of a profile with PPR1 is allowed. If the eUICC already has a profile (regardless of its origin and policy rules associated with it), then the download of a profile with PPR1 is not allowed.

Reviewer 2: ```suggestion
Required. Indicates whether the download of a profile with PPR1 is allowed. If the eUICC already has a profile (regardless of its origin and policy rules associated with it), the download of a profile with PPR1 is not allowed.
```

Reviewer 3: Taking into account the comment about defining the acronym, may I suggest the following addition:
```suggestion
Required. Indicates whether the download of a profile with Profile Policy Rule 1 (PPR1) is allowed. If the eUICC already has a profile (regardless of its origin and policy rules associated with it), the download of a profile with PPR1 is not allowed.
```

Supported operation is Get. Value type is boolean.

<a href="" id="euicc-isactive"></a>**_eUICC_/PPR1AlreadySet**  

> Required. Indicates whether the eUICC has already a profile with PPR1.

Reviewer 2: ```suggestion
Required. Indicates whether the eUICC already has a profile with PPR1.
```

Supported operation is Get. Value type is boolean.

<a href="" id="euicc-profiles"></a>**_eUICC_/DownloadServers**  
Interior node. Represents default SM-DP+ discovery requests.

Supported operation is Get.

<a href="" id="euicc-profiles-iccid"></a>**_eUICC_/DownloadServers/_ServerName_**  

> Interior node. Optional. Node representing the discovery operation for a server name. The node name is the fully qualified domain name of the SM-DP+ server that will be used for profile discovery. Creation of this subtree triggers a discovery request.

Reviewer: Instead of "representing the discovery operation for a server name" I'd suggest "specifying the server name for a discovery operation".

> Interior node. Optional. Node representing the discovery operation for a server name. The node name is the fully qualified domain name of the SM-DP+ server that will be used for profile discovery. Creation of this subtree triggers a discovery request.

Reviewer 2: ```suggestion
Interior node. Optional. Node specifying the server name for a discovery operation. The node name is the fully qualified domain name of the SM-DP+ server that will be used for profile discovery. Creation of this subtree triggers a discovery request.
```

Supported operations are Add, Get, and Delete.

<a href="" id="euicc-profiles-iccid-state"></a>**_eUICC_/DownloadServers/_ServerName_/DiscoveryState**  

> Required. Current state of the discovery operation for the parent ServerName (Requested = 1, Executing = 2, Completed = 3, Failed = 4). Queried by the CSP and only updated by the LPA.

Reviewer: I see that we use this language for other nodes, although we're not consistent. I think I'd prefer that it say it is queried by the MDM rather than by the CSP, but I concede that this is consistent with the description of "eUICC/Profiles/ICCID/State", so perhaps it is best left as-is.

Supported operation is Get. Value type is integer. Default value is 1.

<a href="" id="euicc-profiles-iccid-isenabled"></a>**_eUICC_/DownloadServers/_ServerName_/AutoEnable**  

> Required. Indicates whether the discovered profile must be enabled automatically after install. This must be set by the MDM when the ServerName subtree is created

Reviewer 2: ```suggestion
Required. Indicates whether the discovered profile must be enabled automatically after install. This must be set by the MDM when the ServerName subtree is created.
```
